handsontable/src/3rdparty/walkontable/src/selection/border/border.js

```diff
@@ -48,8 +48,15 @@ class Border {
     this.endStyle = null;
 
     this.cornerDefaultStyle = getCornerStyle(this.instance);
+
     // Offset to moving the corner to be centered relative to the grid.
-    this.cornerCenterPointOffset = -Math.ceil((parseInt(this.cornerDefaultStyle.width, 10) / 2));
+    if (this.wot.wtSettings.getSetting('stylesHandler').isClassicTheme()) {
+      this.cornerCenterPointOffset = -Math.ceil((parseInt(this.cornerDefaultStyle.width, 10) / 2));
```

Comment:
Unnecessary brackets
```suggestion
      this.cornerCenterPointOffset = -Math.ceil(parseInt(this.cornerDefaultStyle.width, 10) / 2);
```